python/views/widget_delegate.py

```diff
@@ -114,6 +118,10 @@ def _create_editor_widget(self, model_index, style_options, parent):
         """
         # the default implementation just calls _create_widget for backwards
         # compatibility.
+
+        if not model_index.isValid():
+            return None
```

Comment:
I tend to always include this check in methods that accept model index args since who knows what kind of craziness can go on in a Qt model, proxy, etc. I'll add a comment here too.